### PR TITLE
feat(YouTube/Overlay buttons): add patch option `WiderButtonsSpace`

### PR DESCRIPTION
**Pull request by MondayNitro**

### CODE REVIEW

**Comment by MondayNitro** [comment]
will it not set bottom margin if widerButtonsSpace is true?
Screenshot_30-9-2024_184941_github com

File: src/main/kotlin/app/revanced/patches/youtube/player/overlaybuttons/OverlayButtonsPatch.kt [hunk 1]
```diff
@@ -198,19 +207,21 @@ object OverlayButtonsPatch : BaseResourcePatch(
                         )
 
                         // Adjust TimeBar and Chapter bottom padding
-                        val timBarItem = mutableMapOf(
-                            "@id/time_bar_chapter_title" to "16.0dip",
-                            "@id/timestamps_container" to "14.0dip"
-                        )
+						if (WiderButtonsSpace == false) {
```
Comment (inotia00) [inline src/main/kotlin/app/revanced/patches/youtube/player/overlaybuttons/OverlayButtonsPatch.kt]: According to the code when the PR was first opened, bottom margin was set only when `WiderButtonsSpace` was FALSE

**Comment by inotia00** [comment]
yes

**Comment by MondayNitro** [comment]
im really sorry i made a mistake, paddingBottom should only be changed if WiderButtonsSpace is false but marginBottom should be applied unconditionally.
sir can you please check when you have time or should i open pr?

**Comment by inotia00** [comment]
No problem, the PR was only released to the dev branch, so I can just reflect this in the next dev build (you don't have to open a PR)

**Comment by MondayNitro** [comment]
thanks🙂

**Comment by inotia00** [comment]
marginBottom: always change
paddingBottom: only change when `WiderButtonsSpace` is FALSE

**Pull request merged**